features/bootstrap/FeatureContext.php

```diff
@@ -1,19 +1,29 @@
 <?php
 
+/*
+ * This file is part of the ScenarioStateBehatExtension project.
+ *
+ * (c) Rodrigue Villetard <rodrigue.villetard@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
 use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\Gherkin\Node\PyStringNode;
 use Symfony\Component\Process\PhpExecutableFinder;
 use Symfony\Component\Process\Process;
 
 /**
- * Behat context class.
+ * @author Rodrigue Villetard <rodrigue.villetard@gmail.com>
```

Comment:
Actually, I’ve originally copy/pasted this file from symfony2 extension
 (https://github.com/Behat/Symfony2Extension/blob/master/features/bootstrap/FeatureContext.php)